api/src/main/java/org/ccci/idm/user/User.java

```diff
@@ -72,8 +72,14 @@
 
     // Global Registry ID
     @Nullable
+    private String grMasterPersonId;
+    @Nullable
+    private String grStageMasterPersonId;
+    @Nullable
+    @Deprecated
```

Comment:
Why are the previous fields being deprecated?

Reply:
Summary of a conversation on this:

grPersonId was originally implemented 2 years ago and we hadn't found any use cases it was actually useful for. The deprecation was meant to cleanup some unused code, but wasn't related to adding master person ID.

There is some interest in utilizing the grPersonId within staffweb authz, so for now we are leaving the attributes present and not deprecating them to hopefully aid with this task.